presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -0,0 +1,197 @@
+* Schema evolution
+
+The Iceberg table state is maintained in metadata files. All changes to table state
+create a new metadata file and replace the old metadata with an atomic swap.
+The table metadata file tracks the table schema, partitioning config,
+custom properties, and snapshots of the table contents.
+
+Iceberg data files can be stored in either Parquet or ORC format, as
+determined by the ``format`` property in the table definition.  The
+table ``format`` defaults to ``ORC``.
+
+Iceberg is designed to improve on the known scalabilty limitations of Hive.  Hive
+stores table metadata in an object store backed by a database like MySQL.  It tracks
+partition locations in the metastore, but not individual data files.  Presto queries
+using the Hive connector must first call the metastore to get partition locations,
+then call the underlying filesystem to list all data files inside each partition,
+and then read metadata from each data file.
+
+Since Iceberg stores the paths to data files in the metadata files, it
+only consults the underlying file system for files that must be read.
+
+Status
+------
+
+The Iceberg connector is not yet supported for production use.
+
+Compatibility With Spark Iceberg
+--------------------------------
+
+Tests have been written to demonstrate that data written by Presto can be read
+using Spark SQL with the Iceberg format, and vice versa.  See
+`TestSparkCompatibility <https://github.com/prestosql/presto/blob/master/presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestSparkCompatibility.java>`_.
+
+Configuration
+-------------
+
+Iceberg supports the same metastore configuration properties as the Hive connector.
+At a minimum, ``hive.metastore.uri`` must be configured:
+
+.. code-block:: none
+
+    connector.name=iceberg
+    hive.metastore.uri=thrift://localhost:9083
+
+Partitioned Tables
+------------------
+
+Iceberg supports partitioning by specifying transforms over the table columns.
+A partition is created for each unique tuple value produced by the transforms.
+Identity transforms are simply the column name. Other transforms are:
+
+===================================== ================================================================================================
+Transform                             Description
+===================================== ================================================================================================
+``year(ts)``                          The partition value is the integer difference in years between ``ts`` and January 1st. 1970.
+
+``month(ts)``                         The partition value is the integer difference in months between ``ts`` and January 1st. 1970.
+
+``day(ts)``                           The partition value is the integer difference in days between ``ts`` and January 1st. 1970.
+
+``hour(ts)``                          The partition value is a TIMESTAMP rounded down to the nearest hour.
```

Comment:
No need to uppercase timestamp here, since it's used as word, not a SQL keyword.

Maybe instead of "rounded down" we could say

> The partition value is a timestamp with the minutes and seconds set to zero

Reply:
Changed.